ci/tests/sanity/sanity_checks_test.py

```diff
@@ -358,6 +367,12 @@ def check_vpool_remove_sanity_test(vpool_name=''):
     if not vpool_name:
         vpool_name = general.test_config.get("vpool", "vpool_name")
 
+    api = Connection.get_connection()
+    vpool_list = api.get_component_by_name('vpools', vpool_name)
+    assert vpool_list, "No vpool found where one was expected"
+    vpool = vpool_list[0]
```

Comment:
vpool = vpool_list[0]
line not needed as you retrieve vpool again on line 385 | 400